lib/OpenQA/Scheduler/Model/Jobs.pm

```diff
@@ -458,7 +458,7 @@ sub _assign_multiple_jobs_to_worker ($self, $jobs, $worker, $directly_chained_jo
     if (my $tmpdir = $worker->get_property('WORKER_TMPDIR')) {
         File::Path::rmtree($tmpdir);
     }
-    my %worker_properties = (JOBTOKEN => random_string(), WORKER_TMPDIR => tempdir());
+    my %worker_properties = (JOBTOKEN => random_string(), WORKER_TMPDIR =>  tempdir(sprintf('worker-%d.XXXXXXXX', $worker_id), TMPDIR => 1));
```

Comment:
Maybe it should also mention the origin service, like `webui-worker-%d.XXXXXXXX` and `scheduler-worker-%d.XXXXXXXX`. After all tracking it down took longer because the error looked like it originated on a worker and not the webui host.

Reply:
done